src/main/java/io/github/jhipster/online/service/GitService.java

```diff
@@ -70,6 +70,8 @@ public void pushNewApplicationToGit(User user, File workingDir, String organizat
         remoteAddCommand.setUri(urIish);
         remoteAddCommand.call();
 
+        String currentBranch = git.getRepository().getFullBranch();
+        if (currentBranch.equals("refs/heads/master")) git.branchRename().setNewName("main").call();
```

Comment:
```suggestion
        if (currentBranch.equals("refs/heads/master")) {
            git.branchRename().setNewName("main").call();
        }
```

Reply:
don't we have prettier:format for Java ?

Reply:
We do, but in our CI we haven't enforced it; I'll enable it in a separate PR. 😄